cmd/explaintest/r/explain_easy.result

```diff
@@ -226,22 +226,20 @@ StreamAgg_12	1.00	root	funcs:sum(col_0)
 explain select 1 in (select c2 from t2) from t1;
 id	count	task	operator info
 Projection_6	10000.00	root	5_aux_0
-└─HashLeftJoin_7	10000.00	root	CARTESIAN left outer semi join, inner:TableReader_14
+└─HashLeftJoin_7	10000.00	root	CARTESIAN left outer semi join, inner:TableReader_13, other cond:eq(1, test.t2.c2)
   ├─IndexReader_11	10000.00	root	index:IndexScan_10
   │ └─IndexScan_10	10000.00	cop	table:t1, index:c2, range:[NULL,+inf], keep order:false, stats:pseudo
-  └─TableReader_14	10.00	root	data:Selection_13
-    └─Selection_13	10.00	cop	eq(1, test.t2.c2)
-      └─TableScan_12	10000.00	cop	table:t2, range:[-inf,+inf], keep order:false, stats:pseudo
+  └─TableReader_13	10000.00	root	data:TableScan_12
+    └─TableScan_12	10000.00	cop	table:t2, range:[-inf,+inf], keep order:false, stats:pseudo
 explain select sum(6 in (select c2 from t2)) from t1;
 id	count	task	operator info
 StreamAgg_12	1.00	root	funcs:sum(col_0)
-└─Projection_22	10000.00	root	cast(5_aux_0)
-  └─HashLeftJoin_21	10000.00	root	CARTESIAN left outer semi join, inner:TableReader_20
+└─Projection_21	10000.00	root	cast(5_aux_0)
+  └─HashLeftJoin_20	10000.00	root	CARTESIAN left outer semi join, inner:TableReader_19, other cond:eq(6, test.t2.c2)
     ├─IndexReader_17	10000.00	root	index:IndexScan_16
```

Comment:
here the `eq(6, test.t2.c2)` is moved into other condition because it is from `6 in (subq)`